webapp/channels/src/components/claim/components/oauth_to_email.tsx

```diff
@@ -99,7 +98,7 @@ const OAuthToEmail = (props: Props) => {
                     />
                 </p>
                 <div className={classNames('form-group', {'has-error': error})}>
-                    <LocalizedInput
+                    <input
                         type='password'
                         className='form-control'
                         name='password'
```

Comment:
could you please pass it here directly instead of creating the a new variable `const placeholderPasswordMessage`
For other variable as well

Reply:
@M-ZubairAhmed changed

Reply:
thank you for this change